### ISSUE TITLE
A11y_RiskAsAServiceUX_RiskSupport_ScreenReader:  NVDA announces extra information while navigating on Dates of Calendar control.

### ISSUE THREAD

**Issue by nirmalsahoo** [nvaccess/nvda#10062]
​Environment Details:​​
Application Name: Risk as a Service UX​
URL: https://dfp.microsoft.com/​
Windows Version: Windows10​​
Chrome Version: 73.0.3683.103 (Official Build) (64-bit)​
NVDA Version: 2019.1.1
​

### Steps to reproduce:
1. Hit the app Url->Navigate to left navigation bar->Expand "Risk Decisioning"->Select "Risk Support".
2. Risk Support page will open.​
3. Start NVDA and navigate to Start/End Date using Tab key and hit enter.
4. Calendar control gets open.
5. Navigate on Dates of Calendar controls using Up/Down arrow keys.
6. Verify NVDA announces extra information for Dates.
### Actual behavior:
Issue1: While navigating on the Dates of Calendar control using up/down arrow keys, NVDA announces some extra information.
For Ex: When NVDA focus is on 12th date, NVDA announces as 
"June 12, 2019 not-selected row4 Wednesday column4
 June 12, 2019 not-selected row-1 Sunday column0".

Issue2: When NVDA focus is on selected Date(June 7) NVDA announces as not selected.

Same issue is observed throughout the application wherever calendar controls are present.
### Expected behavior:
Issue1: NVDA should not announce extra information for Dates.
For Ex: When NVDA focus is on June 12 date NVDA should simply announce as "June 12, 2019 not-selected row4 Wednesday column4".
Issue2: Which Date gets selected NVDA should announce selected for that Date.

 NVDA version:
2019.1.1
 Windows version:
Windows 10
User Impact:​​​
Screen reader users face difficulty to understand correct position(rows/columns) of Dates if Screen reader announces unnecessary extra information for dates.

Recommendations:​​
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
MAS Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
Attachment for reference:
![Issue2](https://user-images.githubusercontent.com/53940787/62778635-85a93900-bace-11e9-8510-2ddcb8eb9e29.png)
![Issue1](https://user-images.githubusercontent.com/53940787/62778636-85a93900-bace-11e9-96e8-6abefc9c3aca.png)

**Comment by DrSooom** [comment]
@nirmalsahoo: Please disable the option "Report object position information" in the Object Presentation NVDA Settings and try again. NVDA should now report less information, but maybe still too much. Am I right? And please reproduce it with the latest Google Chrome version, which should be around 76.x. Thanks.

**Comment by michaelDCurran** [comment]
Closing on request from reporter privately by email.
